.github/workflows/ci.yml

```diff
@@ -82,16 +86,19 @@ jobs:
           ruby-version: .ruby-version
           bundler-cache: true
 
-      - name: Run tests and smoke test seed
-        env:
-          RAILS_ENV: test
-          DATABASE_URL: postgres://postgres:postgres@localhost:5432
+      - name: DB setup and smoke test
         run: |
           bin/rails db:create
           bin/rails db:schema:load
-          bin/rails test
           bin/rails db:seed
 
+      - name: Unit and integration tests
+        run: bin/rails test
+
+      - name: System tests
+        run: DISABLE_PARALLELIZATION=true bin/rails test:system
+        continue-on-error: true # TODO: Eventually we'll enforce for PRs
```

Comment:
As explained in #817, the intention here is to allow for us to start running system tests during the CI process without enforcing them yet.  We'll eventually remove this and require system tests once the app reaches a more stable point.